docs/docsite/rst/win_guide/windows_winrm.rst

```diff
@@ -483,6 +483,8 @@ Kerberos is reliant on a properly-configured environment to
 work. To troubleshoot Kerberos issues, ensure that:
 
 * The hostname set for the Windows host is the FQDN and not an IP address.
+  * If you connect using an IP address you will get the error message `Server not found in Kerberos database`.
+  * To determine if you are connecting using an IP address or an FQDN run your playbook (or call the win_ping module) using the `-vvvv` flag.
```

Comment:
3 v's or above is needed, no need for 4 to show the hostname being connected to.